ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartOptionsEditor.tsx

```diff
@@ -12,12 +12,39 @@
 // limitations under the License.
 
 import { OptionsEditorProps } from '@perses-dev/plugin-system';
-import { Box } from '@mui/material';
+import { Stack, Box } from '@mui/material';
 import { TimeSeriesChartOptions } from './time-series-chart-model';
+import { TimeSeriesChartPanel, TimeSeriesChartProps } from './TimeSeriesChartPanel';
 
 export type TimeSeriesChartOptionsEditorProps = OptionsEditorProps<TimeSeriesChartOptions>;
 
 export function TimeSeriesChartOptionsEditor(props: TimeSeriesChartOptionsEditorProps) {
   const { value } = props;
-  return <Box>{JSON.stringify(value)}</Box>;
+
+  const panelPreviewProps: TimeSeriesChartProps = {
+    definition: {
+      kind: 'Panel',
+      spec: {
+        // TODO: how to exlude display (display.name is required but gets overriden)
+        display: { name: 'Temp Name' },
```

Comment:
Yeah was thinking about this a little too!

If I'm understanding your comment right, my ideas so far were:
- We could have a prop on the panel component to show/hide title
- Build a new component using the underlying TimeSeriesChart + new chrome around it (very minimal chrome)

Reply:
I was thinking about maybe changing up the PanelPlugin API so that the plugin component only gets the `spec` prop and not the entire definition object (we already made this change in both query and variable plugins). Would that help solve it?

Reply:
My idea is for "preview" the PanelDrawer could just render the actual plugin component with the current spec state passed in after we made that change.

Reply:
Yeah not getting the entire definition object would be great! I'll merge this for now with the assumption that using TimeSeriesChartPanel directly in TimeSeriesChartOptionsEditor will change very soon in favor of the solution you both described